features/dependencies.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Install dependencies Tuist.
+
+    Scenario: The project is an application with framework and tests and Dependencies.swift (app_with_framework_and_tests_and_dependencies)
+        Given that tuist is available
+        And I have a working directory
+        Then I copy the fixture app_with_framework_and_tests_and_dependencies into the working directory
+        Then tuist fetches dependencies
```

Comment:
Note that if the file is these acceptance tests are not added to the GitHub [workflow](https://github.com/tuist/tuist/blob/main/.github/workflows/tuist.yml#L58) they won't run on CI.

Reply:
<img width="932" alt="Zrzut ekranu 2020-12-16 o 17 01 38" src="https://user-images.githubusercontent.com/4774319/102374834-172d9280-3fc2-11eb-8bb2-45245cc63139.png">

It is already added. Am I missing something?